lib/remotipart/middleware.rb

```
@@ -9,7 +9,10 @@ def initialize app
 
     def call env
       # Get request params
-      params = Rack::Request.new(env).params
+      begin
+        params = Rack::Request.new(env).params
+      rescue
```

I'm not a fan of silently ignoring errors. Ignore all of them is even worse. Maybe capture the errors that get raised (Bugsnag might help) and use `Rails.logger` to log them, at least.